### PR TITLE
added migrations bundle

### PR DESCRIPTION
**Pull request by ryandillinfelton**
Adding entry for the new migrations bundle in the **SOURCE OF TRUTH** Opening PR early for easier troubleshooting.

### CODE REVIEW

**Review by ryelo — CHANGES_REQUESTED** [review]
https://github.com/RedHatInsights/cloud-services-config/blob/e1f567c18816b852b3122b382d1314bd591381e4/main.yml#L174 needs to be updated

File: main.yml [hunk 1]
```diff
@@ -175,6 +175,21 @@ migration-analytics:
...
   mailing_list: migration-analytics-devel@redhat.com
 
+migrations:
+  title: Migration Services
+  api:
+    versions:
+      - v1
+  deployment_repo: https://github.com/RedHatInsights/xavier-ui-deploy
+  frontend:
+    title: Migration
```
Comment (ryelo) [inline main.yml]: Should this be `Migrations`?

File: main.yml [hunk 2]
```diff
@@ -161,6 +161,20 @@ landing:
...
+migrations:
+  title: Migrations
+  api: 
+    versions:
+      - v1
+  frontend: 
+    title: Migrations
+    sub_apps:
+      - id: migration-analytics
+        title: Migration Analytics
```
Comment (kruai) [inline main.yml]: Supplying a `title` is only necessary if the sub-app isn't defined elsewhere in the config; this field is for things like `/insights/overview`, which aren't a separate app and just act as routing for `/insights`.

File: main.yml [hunk 3]
```diff
@@ -161,6 +161,20 @@ landing:
...
+        group: migrations
```
Comment (kruai) [inline main.yml]: I may be mistaken, but I don't think you need the `group` tag for this one.

File: main.yml [hunk 4]
```diff
@@ -171,7 +185,8 @@ migration-analytics:
   disabled_on_stable: true
   frontend:
     paths:
-      - /staging/migration-analytics
```
Comment (kruai) [inline main.yml:202]: If you're removing it from `/staging`, you should also remove the reference from the Staging bundle.

**Review by kruai — CHANGES_REQUESTED** [review]

File: main.yml (hunk 3 above)
Comment (ryandillinfelton) [reply main.yml]: removed

File: main.yml (hunk 2 above)
Comment (ryandillinfelton) [reply main.yml]: Removed

File: main.yml (hunk 4 above)
Comment (ryandillinfelton) [reply main.yml:202]: Removed reference from staging sub_apps

**Review by kruai — APPROVED** [review]
Looks good now!

**Review by ryelo — APPROVED** [review]

**Pull request merged**